tests/presenters/teaser-presenter.spec.js

```diff
@@ -12,6 +12,21 @@ describe('Teaser Presenter', () => {
 
 	context('classModifiers', () => {
 
+		it('produces an empty array if mods undefined', () => {
+			const content = { mods: undefined };
+			subject = new Presenter(content);
+			expect(subject.classModifiers).to.deep.equal([]);
+		});
+
+		it('does not save previous teaser modifiers to new teaser', () => {
+			const content = { isOpinion: true };
+			let teaser1 = new Presenter(content);
+			let teaser2 =new Presenter(content);
```

Comment:
This doesn't test the condition that we were trying to avoid.

What you want to do is something like;
```
const mods = ['test'];
const content = { mods, isOpinion:true };
const teaser = new Presenter(content);
expect(teaser.classModifiers).to.contain('opinion');
expect(mods).to.not.contain('opinion');